src/terminal/Terminal.zig

```diff
@@ -344,7 +344,7 @@ pub fn print(self: *Terminal, c: u21) !void {
             // VS15 makes it narrow.
             if (c == 0xFE0F or c == 0xFE0E) {
                 // This only applies to emoji
-                const prev_props = unicode.getProperties(prev.cell.content.codepoint);
+                const prev_props = unicode.table.get(prev.cell.content.codepoint);
```

Comment:
`getProperties` (aka `props.get`) now relies on `DisplayWidth`, which we don't pull in as a dependency for the main executable/library. I switched this to use our table data for the properties.